cedar-policy-validator/src/coreschema.rs

```diff
@@ -79,7 +69,7 @@ impl<'a> entities::Schema for CoreSchema<'a> {
     }
 
     fn action_entities(&self) -> Self::ActionEntityIterator {
-        self.actions.values().map(Arc::clone).collect()
+        self.schema.actions.values().map(Arc::clone).collect()
```

Comment:
One last thought. I feel like we should be able to also get rid of this `collect`  and just return `self.schema.actions.values().cloned()`. Would just need to set the right type for `Self::ActionEntityIterator`

Reply:
Addressed in [2f5e11f](https://github.com/cedar-policy/cedar/pull/1290/commits/2f5e11f71425b14b63e4b552b75cfebb99f599d4)